src/components/job-openings/JobOpeningDisplay.vue

```diff
@@ -17,6 +17,23 @@ export default {
       type: Object,
       required: true
     }
+  },
+  methods: {
+    getShortUrlHash() {
+      if (this.jobOpening.JobOpeningShortUrls && this.jobOpening.JobOpeningShortUrls.length > 0) {
```

Comment:
Calling this object hierarchy is cumbersome and long.
Store the array into a temporary variable and evaluate on that
`const shortUrls = this.jobOpening.JobOpeningShortUrls`
`if (shortUrls && shortUrls.length)`